runtime/sema/type.go

```diff
@@ -3241,6 +3242,30 @@ func suggestFixedPointLiteralConversionReplacement(
 	}
 }
 
+var PublicPathConverterType = &FunctionType{
```

Comment:
The code on master has a refactor of using a helper function here (`pathConversionFunctionType`)